code/__DEFINES/access.dm

```diff
@@ -1,148 +1,183 @@
+#define ACCESS_TCOMMS 46
 /// Room and launching.
-#define ACCESS_AUX_BASE 72
-/// Service access, for service hallway and service consoles
-#define ACCESS_SERVICE 73
+#define ACCESS_AUX_BASE 47
+#define ACCESS_EXTERNAL_AIRLOCKS 48
+///ID 49-54 currently unused
+/// ENGINEERING - JOBS
+#define ACCESS_CE 55
+///ID 56-59 currently unused
 
-	//BEGIN CENTCOM ACCESS
-	/*Should leave plenty of room if we need to add more access levels.
-	Mostly for admin fun times.*/
-/// General facilities. CentCom ferry.
-#define ACCESS_CENT_GENERAL 101
-/// Thunderdome.
-#define ACCESS_CENT_THUNDER 102
-/// Special Ops. Captain's display case, Marauder and Seraph mechs.
-#define ACCESS_CENT_SPECOPS 103
-/// Medical/Research
-#define ACCESS_CENT_MEDICAL 104
-/// Living quarters.
-#define ACCESS_CENT_LIVING 105
-/// Generic storage areas.
-#define ACCESS_CENT_STORAGE 106
-/// Teleporter.
-#define ACCESS_CENT_TELEPORTER 107
-/// Captain's office/ID comp/AI.
-#define ACCESS_CENT_CAPTAIN 109
-/// The non-existent CentCom Bar
-#define ACCESS_CENT_BAR 110
+/// - - - MEDICAL 60-79 - - -
+/// Medical general access
+#define ACCESS_MEDICAL 60
+#define ACCESS_MORGUE 61
+/// Pharmacy access (Chemistry room in Medbay)
+#define ACCESS_PHARMACY 62
+#define ACCESS_SURGERY 63
+///Allows access to chemistry factory areas on compatible maps
+#define ACCESS_PLUMBING 64
+///ID 65-74 currently unused
+/// MEDICAL - JOBS
+#define ACCESS_CMO 75
+#define ACCESS_VIROLOGY 76
+#define ACCESS_PSYCHOLOGY 77
+///ID 78-79 currently unused
+
+/// - - - SUPPLY 80-99 - - -
+///Cargo general access
+#define ACCESS_CARGO 80
+#define ACCESS_MAIL_SORTING 81
+/// For releasing minerals from the ORM
+#define ACCESS_MINERAL_STOREROOM 82
+#define ACCESS_MINING_STATION 83
+#define ACCESS_VAULT 84
+///ID 85-94 currently unused
+/// SUPPLY - JOBS
+#define ACCESS_QM 95
+#define ACCESS_MINING 96
+///ID 97-99
 
-	//The Syndicate
-/// General Syndicate Access. Includes Syndicate mechs and ruins.
-#define ACCESS_SYNDICATE 150
-/// Nuke Op Leader Access
-#define ACCESS_SYNDICATE_LEADER 151
+/// - - - SCIENCE 100-119 - - -
+///Science general access
+#define ACCESS_SCIENCE 100
+#define ACCESS_RESEARCH 101
+#define ACCESS_ORDNANCE_STORAGE 102
+///ID 103-114 currently unused
+/// SCIENCE - JOBS
+#define ACCESS_RD 115
+#define ACCESS_GENETICS 116
+#define ACCESS_ROBOTICS 117
+#define ACCESS_ORDNANCE 118
+#define ACCESS_XENOBIOLOGY 119
 
-	//Away Missions or Ruins
-	/*For generic away-mission/ruin access. Why would normal crew have access to a long-abandoned derelict
+/// - - - SERVICE 120-139 - - -
+///Service general access
+#define ACCESS_SERVICE 120
+#define ACCESS_THEATRE 121
+#define ACCESS_CHAPEL_OFFICE 122
+#define ACCESS_CREMATORIUM 123
+#define ACCESS_LIBRARY 124
+#define ACCESS_BAR 125
+#define ACCESS_KITCHEN 126
+#define ACCESS_HYDROPONICS 127
+///ID 128-134 currently unused
+/// SERVICE - JOBS
+///ID 135 reserved for HOP in case it changes
+#define ACCESS_JANITOR 136
+#define ACCESS_LAWYER 137
```

Comment:
why are these two separated?

Reply:
The janitor also has access to the trash room, and the lawyer is also specialized to other areas. Technically you could rename ACCESS_LAWYER to ACCESS_LAW_OFFICE and move it down but I've tried to touch the names as little as possible I'm thinking about later on removing court access to make it sec entrance since it's kind of unnecessary. When I do that I will probably change the name and move it down.